modules/olm-installing-specific-version-cli.adoc

```diff
@@ -30,9 +30,131 @@ endif::[]
 
 .Procedure
 
+. Look up the available versions and channels of the Operator you want to install by running the following command:
++
+.Command syntax
+[source,terminal]
+----
+$ oc describe packagemanifests <operator_name> -n <catalog_namespace>
+----
++
+For example, the following command prints the available channels and versions of the Red Hat Quay Operator from OperatorHub:
++
+.Example command
+[source,terminal]
+----
+$ oc describe packagemanifests quay-operator -n openshift-marketplace
+----
++
+.Example output
+[%collapsible]
+====
+[source,text]
+----
+Name:         quay-operator
+Namespace:    operator-marketplace
+Labels:       catalog=redhat-operators
+              catalog-namespace=openshift-marketplace
+              hypershift.openshift.io/managed=true
+              operatorframework.io/arch.amd64=supported
+              operatorframework.io/os.linux=supported
+              provider=Red Hat
+              provider-url=
+Annotations:  <none>
+API Version:  packages.operators.coreos.com/v1
+Kind:         PackageManifest
+...
+    Current CSV:  quay-operator.v3.7.11
+...
+    Entries:
+      Name:       quay-operator.v3.7.11
+      Version:    3.7.11
+      Name:       quay-operator.v3.7.10
+      Version:    3.7.10
+      Name:       quay-operator.v3.7.9
+      Version:    3.7.9
+      Name:       quay-operator.v3.7.8
+      Version:    3.7.8
+      Name:       quay-operator.v3.7.7
+      Version:    3.7.7
+      Name:       quay-operator.v3.7.6
+      Version:    3.7.6
+      Name:       quay-operator.v3.7.5
+      Version:    3.7.5
+      Name:       quay-operator.v3.7.4
+      Version:    3.7.4
+      Name:       quay-operator.v3.7.3
+      Version:    3.7.3
+      Name:       quay-operator.v3.7.2
+      Version:    3.7.2
+      Name:       quay-operator.v3.7.1
+      Version:    3.7.1
+      Name:       quay-operator.v3.7.0
+      Version:    3.7.0
+    Name:         stable-3.7
+...
+   Current CSV:  quay-operator.v3.8.5
+...
+   Entries:
+      Name:         quay-operator.v3.8.5
+      Version:      3.8.5
+      Name:         quay-operator.v3.8.4
+      Version:      3.8.4
+      Name:         quay-operator.v3.8.3
+      Version:      3.8.3
+      Name:         quay-operator.v3.8.2
+      Version:      3.8.2
+      Name:         quay-operator.v3.8.1
+      Version:      3.8.1
+      Name:         quay-operator.v3.8.0
+      Version:      3.8.0
+    Name:           stable-3.8
+  Default Channel:  stable-3.8
+  Package Name:     quay-operator
+----
+====
++
+[TIP]
+====
+You can format an Operator's version and channel information in the YAML format by running the following command:
+
+[source,terminal]
+----
+$ oc get packagemanifests <operator_name> -n <operator_namespace> -o yaml
```

Comment:
@michaelryanpeter 
1, it should be `oc get packagemanifests <operator_name> -n <catalog_namespace> -o yaml`, not `oc get packagemanifests <operator_name> -n <operator_namespace> -o yaml`

2, if there are multiple catalog sources which include same operator in one namespace, `oc get packagemanifests <operator_name> -n <catalog_namespace> -o yaml` will return packagemanifests random among these two catalog source (it is known issue).  if we meet such situation and know the catalog source we want to use, the command should be `oc get packagemanifest --selector=catalog=<catalogsource_name> --field-selector metadata.name=<operator_name> -n <catalog_namespace> -o yaml`.
note: it returns the list which only includes one item. 
frankly it is same to `packagemanifests <operator_name> -n <catalog_namespace>`, but I do not find the good way to get it because `oc describe` does not support `--field-selector`

Reply:
@kuiwang02 

1. I corrected variable name to `<catalog_namespace>`
2. In almost all cases, we document known issues and workarounds in the release notes. We do this to avoid cluttering up the core documentation with stale workarounds for bugs after they are fixed. I created a ticket to track the update to the 4.13 release notes with the known issue and workaround: OCPBUGS-12151. I added a couple of questions for you in the Jira. 

Would you please take another look at the PR when you get the chance? Thank you!

Reply:
ok. got it.
I am ok for the doc with known issue.

Reply:
I will start a conversation on Slack where we can talk with engineering about how to document the known issue. I will make sure you are included in the thread.

Do you need any other changes to this PR in order to approve it for 4.13 GA? Thank you!

Reply:
no other comment. thanks